cgroups/fs/apply_raw.go

```diff
@@ -58,7 +58,7 @@ type subsystem interface {
 	// Removes the cgroup represented by 'data'.
 	Remove(*data) error
 	// Creates and joins the cgroup represented by data.
-	Set(*data) error
+	Apply(*data) error
```

Comment:
`Join` is part of work,
`Apply` = `Join` + `Set`, so I think Apply would be better.

PS: I'll add `Set` interface and separate `Apply` to `Join` and `Set` in a subsequent PR.